blog/_posts/2017-08-14-with-love-team-bundledore-india.md

```diff
@@ -0,0 +1,73 @@
+title: With love, Team Bundledore, India !
+ layout: post
+ created_at: Saturday August 14 2017
+ permalink: blog/2017-08-14-bundledore-with-love-team-Bundledore-India
+ current: blog
+ author: Bundledore
+ twitter: TeamBundledore
+ categories:student-posts
+ 
+ 
+ ### Hello, people! 
+ A very Happy Independence Day to all my fellow Indians here :) We are Team Bundledore, and our project is [AFDC League Management System](https://leagues.afdc.com/) 
+ 
+ The last one and a half-month has been crazy hectic with all the learning, connecting and mentoring. We have truly never enjoyed an internship so much before :)
```

Comment:
I know that the word "crazy" seems to be a very popular adjective in the English language, but at the same time it may leave some people uncomfortable and there's usually a more precise word to use than "crazy". 🙂

I think in this case the word "hectic" illustrates very well what the past month and a half looked like, so "crazy" could be dropped or replaced by "very".

This article explains the origin of the word and why it may be considered ableist (just ignore the US-centric part about the first amendment 😄). http://www.xojane.com/issues/cancel-crazy